src/Microsoft.Fast.Components.FluentUI/Components/FluentHorizontalScroll.razor

```diff
@@ -0,0 +1,15 @@
+<fluent-horizontal-scroll speed=@Speed
+                          easing=@Easing
+                          @attributes="AdditionalAttributes">
+    @ChildContent
+</fluent-horizontal-scroll>
+@code
+{
+    [Parameter] public int? Speed { get; set; }
+
+    [Parameter] public string? Easing { get; set; }
```

Comment:
Should we add some documentation here that indicates what the options are?

Reply:
Or maybe introduce an enum for `Easing`?

Reply:
And perhaps it is better to make `Speed` not nullable and give it a default value (of 600?).